🐛 middleware monitor error

**Fiber version**
v2.0.6
**Issue description**
server shutdown with errors when run wrk -t 4 -c 400 -d 20s http://localhost:8091/test:

panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x4691427]

goroutine 34 [running]:
github.com/gofiber/fiber/v2/middleware/monitor.updateStatistics(0xc00016dc70)
	/Users/zerone/go/pkg/mod/github.com/gofiber/fiber/v2@v2.0.6/middleware/monitor/monitor.go:98 +0x127
github.com/gofiber/fiber/v2/middleware/monitor.New.func1.1(0xc00016dc70)
	/Users/zerone/go/pkg/mod/github.com/gofiber/fiber/v2@v2.0.6/middleware/monitor/monitor.go:61 +0x2b
created by github.com/gofiber/fiber/v2/middleware/monitor.New.func1
	/Users/zerone/go/pkg/mod/github.com/gofiber/fiber/v2@v2.0.6/middleware/monitor/monitor.go:59 +0xbf
exit status 2

**Code snippet**

```go
package main

import "github.com/gofiber/fiber/v2"
import "github.com/gofiber/fiber/v2/middleware/monitor"


func main() {
  app := fiber.New()

  // Steps to reproduce
 app.Get("/dashboard", monitor.New())
 app.Get("/test", func(c *fiber.Ctx) error {
	 return c.SendString("helloweb")
 })

  log.Fatal(app.Listen(":8091"))
}
```

--- a/middleware/monitor/monitor.go
+++ b/middleware/monitor/monitor.go
@@ -91,14 +91,17 @@ func updateStatistics(p *process.Process) {
 	pidCpu, _ := p.CPUPercent()
 	monitPidCpu.Store(pidCpu / 10)
 
-	osCpu, _ := cpu.Percent(0, false)
-	monitOsCpu.Store(osCpu[0])
+	if osCpu, _ := cpu.Percent(0, false); len(osCpu) > 0 {
+		monitOsCpu.Store(osCpu[0])
+	}
 
-	pidMem, _ := p.MemoryInfo()
-	monitPidRam.Store(pidMem.RSS)
+	if pidMem, _ := p.MemoryInfo(); pidMem != nil {
+		monitPidRam.Store(pidMem.RSS)
+	}
 
-	osMem, _ := mem.VirtualMemory()
-	monitOsRam.Store(osMem.Used)
+	if osMem, _ := mem.VirtualMemory(); osMem != nil {
+		monitOsRam.Store(osMem.Used)
+	}
 
 	pidConns, _ := net.ConnectionsPid("tcp", p.Pid)
 	monitPidConns.Store(len(pidConns))

